### PR TITLE
Remove outdated namespace deletion information

### LINKED ISSUE #652: Remove debugging-deletes.md
`debugging-deletes.md` shouldn't be relevant anymore, as we're now two releases past the version that had problems, and we've added code to remove the finalizer that caused the problem.  Remove and re-gen the docs without it.

### PR DESCRIPTION
**Pull request by nrb**
Fixes #652 

Signed-off-by: Nolan Brubaker <nolan@heptio.com>

### CODE REVIEW

File: v0.9.0/troubleshooting.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ These tips can help you troubleshoot known issues. If they don't help, you can [
 
 * [Debug restores][1]
 
-[0]: debugging-deletes.md
```
Comment (skriss) [inline v0.9.0/troubleshooting.md]: this is still referenced above, need to remove the ref

**Comment by Bradamant3** [comment]
Also need to update the TOC

**Comment by nrb** [comment]
@skriss Removed from master and v0.9.0 docs

**Comment by skriss** [comment]
Fixes #652

**Comment by skriss** [comment]
LGTM

**Pull request merged**